addons/mail/static/src/discuss/call/common/call_settings.js

```diff
@@ -36,6 +33,10 @@ export class CallSettings extends Component {
         });
     }
 
+    get isCompact() {
+        return this.env.inChatWindow;
+    }
```

Comment:
`isCompact` here is redundant with condition in `thread_action`. Better improve locality of concern.

What I suggest is to turn it into an (optional) prop, which is passed by `thread_action`. That way the conditional of thread action and passing this prop are exactly at the same place.